src/SPC/util/PkgConfigUtil.php

```diff
@@ -39,18 +39,23 @@ public static function getLibsArray(string $pkg_config_str, bool $force_short_na
         $libs = explode(' ', trim($result));
 
         // get other things
-        $result = self::execWithResult("pkg-config --static --libs --libs-only-other {$pkg_config_str}");
+        $result = self::execWithResult("pkg-config --static --libs-only-other {$pkg_config_str}");
         // convert libxxx.a to -L{path} -lxxx
         $exp = explode(' ', trim($result));
         foreach ($exp as $item) {
+            if (str_starts_with($item, '-L')) {
+                $libs[] = $item;
+                continue;
+            }
             // if item ends with .a, convert it to -lxxx
-            if (str_ends_with($item, '.a') && str_starts_with($item, 'lib') && $force_short_name) {
+            if (str_ends_with($item, '.a') && (str_starts_with($item, 'lib') || str_starts_with($item, BUILD_LIB_PATH))) {
```

Comment:
necessary for -lz in libcrypto.pc, as it contains `BUILD_LIB_PATH/pkgconfig/../../lib/libz.a`